lua/utils/init.lua

```diff
@@ -88,11 +88,16 @@ function utils.toggle_autoformat()
 end
 
 function utils.reload_lv_config()
+  -- Hot reload of statusline
   require("core.lualine").config()
-  vim.cmd "source ~/.local/share/lunarvim/lvim/lua/settings.lua"
+
+  -- Remove package cache
+  package.loaded["settings"] = nil
```

Comment:
Also not necessary + this does nothing since no code is evaluated at module loading time.
IMO, `require "settings"` is not useful

Reply:
Agreed